linux_os/guide/system/network/network-wireless/wireless_software/kernel_module_cfg80211_disabled/rule.yml

```diff
@@ -0,0 +1,39 @@
+documentation_complete: true
+
+prodtype: fedora,rhcos4,rhel8,rhel9
+
+title: 'Disable Wireless Kernel cfg80211 Module'
+
+description: |-
+    The kernel's module loading system can be configured to prevent
```

Comment:
The description states something different than the rule actually does. The rule (thanks to the platform) disables only one module - cfg80211.
I suggest you use this macro:
    {{{ describe_module_disable(module="cfg80211" }}}
You can review the macro here: https://github.com/ComplianceAsCode/content/blob/master/shared/macros.jinja#L528
Please fix this for all rules you create.